roles/confluent.variables/vars/main.yml

```
@@ -641,19 +721,43 @@ kafka_rest_properties:
                             'confluent.monitoring.interceptor.', kafka_rest_truststore_path, kafka_rest_truststore_storepass, kafka_rest_keystore_path, kafka_rest_keystore_storepass, kafka_rest_keystore_keypass,
                             false, sasl_plain_users.kafka_rest.principal, sasl_plain_users.kafka_rest.password, sasl_scram_users.kafka_rest.principal, sasl_scram_users.kafka_rest.password,
                             kerberos_kafka_broker_primary|default('kafka'), kafka_rest_keytab_path, kafka_rest_kerberos_principal|default('rp'),
-                            kafka_rest_ldap_user, kafka_rest_ldap_password, mds_urls) }}"
+                            kafka_rest_ldap_user, kafka_rest_ldap_password, mds_bootstrap_server_urls) }}"
   rbac:
     enabled: "{{rbac_enabled}}"
     properties:
       kafka.rest.resource.extension.class: io.confluent.kafkarest.security.KafkaRestSecurityResourceExtension
       rest.servlet.initializor.classes: io.confluent.common.security.jetty.initializer.InstallBearerOrBasicSecurityHandler
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       client.confluent.metadata.server.urls.max.age.ms: 60000
-      confluent.metadata.bootstrap.server.urls: "{{mds_urls}}"
+      confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
       confluent.metadata.server.urls.max.age.ms: 60000
       confluent.metadata.basic.auth.user.info: "{{kafka_rest_ldap_user | default('rest') }}:{{kafka_rest_ldap_password | default('pass')}}"
       confluent.metadata.http.auth.credentials.provider: BASIC
 
+  rbac_external_client:
+    enabled: "{{rbac_enabled and external_mds_enabled and mds_broker_listener.ssl_enabled }}"
```

```suggestion
    enabled: "{{rbac_enabled and external_mds_enabled and mds_ssl_enabled }}"
```
the sr properties below are for connecting to sr, not to the mds server, i dont think they should come into play here
monitoring interceptor is for connecting to current clusters kafka, not mds

---

I understand that, however it doesn't not work without all TLS properties being set.  I went through this with schema registry as well, whereby if I left any out, the service would fail to start correctly.  I can go back and test again to double check if I can remove any of these, but it was just easier to include them and be safe.